ledger_device_ui_sdk/examples/gadgets.rs

```diff
@@ -0,0 +1,133 @@
+#![no_std]
+#![no_main]
+
+use core::panic::PanicInfo;
+#[panic_handler]
+fn panic(_: &PanicInfo) -> ! {
+    loop {}
+}
+
+use nanos_sdk::buttons::*;
+use nanos_ui::layout::{Layout, Location, StringPlace};
+use nanos_ui::ui;
```

Comment:
crate names should be renamed?
do we have a CI that check that it build?

Reply:
no CI yet, but I forgot to update the crate names...

Reply:
I think that CI integration should be integrated in a next step/PR...

Reply:
agreed